backend/Squirrel.Core/Squirrel.Core.WebAPI/Program.cs

```diff
@@ -49,9 +50,17 @@
 
 app.UseCors(opt => opt
     .AllowAnyHeader()
+    .WithExposedHeaders("Content-Disposition")
     .AllowAnyMethod()
     .AllowAnyOrigin());
 
+app.UseStaticFiles();
+app.UseStaticFiles(new StaticFileOptions()
```

Comment:
Do you need it twice? 
+ I don't think that you need it in controller and here separately